CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# CHANGELOG
+
+## 3.0.0
+#### New Functionality
+
+- [#543](https://github.com/marklogic/node-client-api/issues/543) - Node-client-api now prefers ipv4 over ipv6
+- [#562](https://github.com/marklogic/node-client-api/issues/562) - CtsQueryBuilder is available
+- [#565](https://github.com/marklogic/node-client-api/issues/565 ) - Node-client-api now uses Node - 14
```

Comment:
Maybe upgrading to Node.js 14 belongs under the improvement heading instead of under new functionality?

(Given that the API was using Node.js before.)